packages/schematics/angular/library/index.ts

```diff
@@ -133,7 +117,29 @@ export default function (options: GenerateLibraryOptions): Rule {
         mergeWith(templateSource),
       ])),
       options.skipPackageJson ? noop() : addDependenciesAndScriptsToPackageJson(),
-      options.skipTsConfig ? noop() : updateTsConfig(name, forwardSlashes(entryFilePath)),
+      schematic('module', {
+        name: name,
+        commonModule: false,
+        flat: true,
+        path: `/${name}/src`,
+        sourceDir: options.baseDir,
```

Comment:
`sourceDir` was removed.  The path is relative to the workspace root.

Reply:
I fixed that one in another commit in this PR AFAIK.